ClothesApp/ClothesApp/Program.cs

```diff
@@ -1 +1,225 @@
-﻿Console.WriteLine("Hello");
\ No newline at end of file
+﻿using System.Collections;
+using ClothesApp;
+using ClothesApp.Entities;
+using Microsoft.EntityFrameworkCore;
+
+using var db = new ClothesAppContext();
+
+Console.WriteLine("-----------------LAZY-----------------------");
+Console.WriteLine("get_products_by_brand_id");
+var products = await db.Products
+    .Where(p => p.BrandId == 1)
+    .ToListAsync();
+    
+Console.WriteLine("after query string");
+
+foreach (var p in products)
+{
+    Console.WriteLine($"id: {p.Id}, " +
+                      $"brand_id: {p.BrandId}, " +
+                      $"brand_name: {p.Brand.Name}, " +
+                      $"category_id: {p.CategoryId}," +
+                      $"name: {p.Name}," +
+                      $"price: {p.Price}," +
+                      $"quantity: {p.Quantity}," +
+                      $"created at: {p.CreatedAt}");
+}
+    
+Console.WriteLine("---------------------------------------------------------");
+    
+Console.WriteLine("get_brands_with_products_amount");
+var brands = await db.Products
+    .GroupBy(b => b.BrandId, (b, p) => 
+        new
+        {
+            BrandId = b,
+            ProductCount = p.Count()
+        })
+    .OrderByDescending(p => p.ProductCount)
+    .ToListAsync();
+
+Console.WriteLine("after query string");
+    
+foreach(var b in brands)
+{
+    Console.WriteLine($"brand_id: {b.BrandId}, " +
```

Comment:
I think there will be no lazy loading as nothing is fetched from child entities

Reply:
This query uses 1 table. There is no need to refer to child entities